lib/broccoli/ember-app.js

```diff
@@ -904,7 +904,11 @@ class EmberApp {
 
     // TODO: This isn't quite correct (but it does function properly in most cases),
     // and should be re-evaluated before enabling the `MODULE_UNIFICATION` feature
-    this._srcAfterStylePreprocessing = preprocessCss(srcAfterPreprocessTreeHook, '/src/ui/styles', '/assets', options);
+    let srcStylesPath = this._resolveLocal('src/ui/styles');
+
+    if (existsSync(srcStylesPath)) {
+      this._srcAfterStylePreprocessing = preprocessCss(srcAfterPreprocessTreeHook, '/src/ui/styles', '/assets', options);
```

Comment:
I'm unsure about this exists check. The check prevents some currently supported use cases (e.g. having src/ui/styles/app.scss auto-generated by an earlier preprocessor). 

Can you explain what this change accomplishes (independent of the other change)?